src/VSIX/source.extension.vsixmanifest

```diff
@@ -1,42 +1,36 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
-  <Metadata>
-    <Identity Id="Npgsql.VSIX.07074491-d0c6-4138-9d44-ea3f8920e84b" Version="4.1.12" Language="en-US" Publisher="npgsql" />
-    <DisplayName>Npgsql PostgreSQL Integration</DisplayName>
-    <Description xml:space="preserve">Allows you to connect to PostgreSQL from within Visual Studio via Server Explorer, create EF6 model from existing database, etc.</Description>
-    <MoreInfo>https://www.npgsql.org/doc/ddex.html</MoreInfo>
-    <License>Resources\License.txt</License>
-    <Icon>Resources\PostgreSQL.ico</Icon>
-    <Tags>npgsql;postgresql;postgres;database;ddex;entity framework</Tags>
-  </Metadata>
-  <Installation>
-    <InstallationTarget Version="[14.0,17.0)" Id="Microsoft.VisualStudio.Community">
-      <ProductArchitecture>x86</ProductArchitecture>
-    </InstallationTarget>
-    <InstallationTarget Version="[14.0,17.0)" Id="Microsoft.VisualStudio.Pro">
-      <ProductArchitecture>x86</ProductArchitecture>
-    </InstallationTarget>
-    <InstallationTarget Version="[14.0,17.0)" Id="Microsoft.VisualStudio.Enterprise">
-      <ProductArchitecture>x86</ProductArchitecture>
-    </InstallationTarget>
-
-    <InstallationTarget Version="[17.0,18.0)" Id="Microsoft.VisualStudio.Community">
-      <ProductArchitecture>amd64</ProductArchitecture>
-    </InstallationTarget>
-    <InstallationTarget Version="[17.0,18.0)" Id="Microsoft.VisualStudio.Pro">
-      <ProductArchitecture>amd64</ProductArchitecture>
-    </InstallationTarget>
-    <InstallationTarget Version="[17.0,18.0)" Id="Microsoft.VisualStudio.Enterprise">
-      <ProductArchitecture>amd64</ProductArchitecture>
-    </InstallationTarget>
-  </Installation>
-  <Dependencies>
-    <Dependency Id="Microsoft.Framework.NDP" DisplayName="Microsoft .NET Framework" d:Source="Manual" Version="[4.5,)" />
-  </Dependencies>
-  <Assets>
-    <Asset Type="Microsoft.VisualStudio.VsPackage" d:Source="Project" d:ProjectName="%CurrentProject%" Path="|%CurrentProject%;PkgdefProjectOutputGroup|" />
-  </Assets>
-  <Prerequisites>
-    <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,)" DisplayName="Visual Studio core editor" />
-  </Prerequisites>
+    <Metadata>
```

Comment:
Would be better not to change indentation here to now show all lines as "changed"

Reply:
I am very sorry about it; I autoformat all files on close.

Although, there is no .editorconfig file in the codebase; it could solve problems like this. See: [Code style preferences](https://docs.microsoft.com/en-us/visualstudio/ide/code-styles-and-code-cleanup?view=vs-2022)

Reply:
No need to be sorry :)

Yeah, there's an .editorconfig in more recent versions (4.1 is really old!).